### DISCUSSION TITLE
Just taking a look at something cool: Statebus

### DISCUSSION THREAD

**Discussion by aykaramba** [rabbibotton/clog#288] [Ideas]
I don't fully understand this technology, however, a lot of what I see in the video on their home page reminds me of what is going on in Clog.  What does everyone think of Statebus and how it maps on to / along side Clog framework ideas?  What can be learned from Statebus?

https://stateb.us/

**Comment by rabbibotton** [comment]
Took a quick look. It is like the clog-connect layer but for node instead
of CL.

**Reply by aykaramba** [reply]
Neat.  As always, way ahead of the curve rabbi.
